site/scripts/prepare-blocks.ts

```diff
@@ -489,14 +495,44 @@ const script = async () => {
       });
 
       const blockMetadata = await fs.readJson(blockMetadataPath);
-      blockMetadata.unstable_hubInfo = {
-        ...hubInfo,
-        checksum: blockInfoChecksum,
-        commit: blockInfo.commit,
-        preparedAt: new Date().toISOString(),
-        name: blockInfo.name,
+
+      const exampleGraphFileExists = await fs.pathExists(
+        blockMetadataPath.replace("block-metadata.json", "example-graph.json"),
+      );
+
+      const blockDistributionFolderUrl = `${FRONTEND_URL}/blocks/${blockName}`;
+
+      const now = new Date().toISOString();
+
+      const extendedBlockMetadata: ExpandedBlockMetadata = extendBlockMetadata({
+        metadata: blockMetadata,
+        source: {
+          blockDistributionFolderUrl,
+          pathWithNamespace: blockName,
+          repoCommit: blockInfo.commit,
+          repoDirectory: blockInfo.folder,
+          repository: blockInfo.repository,
+        },
+        timestamps: {
+          lastUpdated: now,
+        },
+        includesExampleGraph: exampleGraphFileExists,
+      });
+
+      const blockMetadataToWrite: BlockMetadataOnDisk = {
+        ...extendedBlockMetadata,
+        unstable_hubInfo: {
```

Comment:
add all the extended metadata here so we have one consistent set of extended metadata everywhere we make it available